### PR TITLE
Addtocart

### PR DESCRIPTION
**Pull request by preethasri**
created add to cart feature

livelink:https://deploy-preview-5--ornatestore.netlify.app/addtocart/addtocart.html

### CODE REVIEW

**Review by BelideNikhil — APPROVED** [review]
Try Saving the color as a variable.
Also use REM instead of px for margin and padding

File: addtocart/addtocart.css [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  justify-content: space-between;
+  background-color: #f1f1f1;
+  padding: 10px;
+  margin-bottom: 20px;
+}
+.nav-links {
+    padding-top: 20px;
+}
+
+.nav-links a {
```
Comment (msdeshmukh009) [inline addtocart/addtocart.css:23]: Maybe you can assign this class directly to the anchor links.

File: addtocart/addtocart.css [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    --product-name: #ff2020;
+    --primary-color: #e72744;
+    --secondary-color: #F05524;
+    --primary-btn: #6495ED;
+}
+/* navbar */
+.landing-page-nav{
+  display:flex;
+  justify-content: space-between;
+  background-color: #f1f1f1;
```
Comment (msdeshmukh009) [inline addtocart/addtocart.css:15]: Color variable maybe?

File: addtocart/addtocart.css [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+  
+}
+.search-bar{
+  max-width:800px;
+  width:100%;
+  
+}
+.form{
+  display:flex;
+
```
Comment (msdeshmukh009) [inline addtocart/addtocart.css:47]: Use code formatter to remove this empty space.

File: addtocart/addtocart.css [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+  padding:5px;
+}
+.product-details{
+   padding:5px 18px;
+}
+.product-details span{
+  display:inline-block;
+   padding:5px 18px;
+  
+
```
Comment (msdeshmukh009) [inline addtocart/addtocart.css:102]: remove this space as well.

File: addtocart/addtocart.css [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+}
+.item-price,.discount,.delivery-charges,.total-amount{
+  display: flex;
+  justify-content: space-between;
+}
+.final-details{
+  margin-left:8rem;
+  
+}
+hr{
```
Comment (msdeshmukh009) [inline addtocart/addtocart.css:154]: Maybe create a class for this.

**Review by msdeshmukh009 — APPROVED** [review]
This is looking good but I will suggest you use some code formatter like prettier which will take care of empty spaces in the code.

File: addtocart/addtocart.css (hunk 5 above)
Comment (preethasri) [reply addtocart/addtocart.css:154]: thank you mahesh for valuable review.will  refactor all  this once i finish my  assignment to submit

**Pull request merged**